diff -urN glunarclock-0.32.4.vanilla/screen-exec/Makefile.in glunarclock-0.32.4/screen-exec/Makefile.in
--- glunarclock-0.32.4.vanilla/screen-exec/Makefile.in	2005-04-12 16:40:13.000000000 +0200
+++ glunarclock-0.32.4/screen-exec/Makefile.in	2006-06-03 22:59:05.000000000 +0200
@@ -216,7 +216,6 @@
 	-DG_DISABLE_DEPRECATED		\
 	-DGDK_DISABLE_DEPRECATED	\
         -DGTK_DISABLE_DEPRECATED	\
-	-DGNOME_DISABLE_DEPRECATED	\
 	$(NULL)
 
 
diff -urN glunarclock-0.32.4.vanilla/screen-exec/egg-screen-help.c glunarclock-0.32.4/screen-exec/egg-screen-help.c
--- glunarclock-0.32.4.vanilla/screen-exec/egg-screen-help.c	2005-04-10 18:01:43.000000000 +0200
+++ glunarclock-0.32.4/screen-exec/egg-screen-help.c	2006-06-03 22:59:05.000000000 +0200
@@ -30,6 +30,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <glib.h>
+#include <glib/gi18n.h>
 #include <libgnome/gnome-help.h>
 #include <libgnome/gnome-program.h>
 #include <libgnome/gnome-i18n.h>
